charts/images.yaml

```diff
@@ -112,8 +112,9 @@ images:
   repository: alpine
   tag: "3.10.3"
 - name: alpine-iptables
+  sourceRepository: github.com/DockToFuture/alpine-iptables
```

Comment:
Shouldn't this image Dockerfile go in a repo under gardener org and to be released by ci/cd, not manually? For example gardener/vpn?